sdk/src/index.ts

```diff
@@ -0,0 +1,25 @@
+import dotenv from "dotenv";
+import { Invariant } from "./invariant.js";
+import { getDeploymentData, initPolkadotJs } from "./utils.js";
+dotenv.config();
+
+(async function main() {
```

Comment:
This convention of top-level function seems to be legacy: `(async function main() {})();`
Try this way: https://github.com/invariant-labs/psp22-ts/blob/main/src/index.ts